docs/manuals/source/TasksAndConcepts/MigrationAndCopy.rst

```diff
@@ -32,7 +32,7 @@ Jobs can be selected for migration based on a number of criteria such as:
 
 The details of these selection criteria will be defined below.
 
-To run a Migration job, you must first define a Job resource very similar to a Backup Job but with :config:option:`dir/job/Type`\  = Migrate instead of :config:option:`dir/job/Type`\  = Backup. One of the key points to remember is that the Pool that is specified for the migration job is the only pool from which jobs will be migrated, with one exception noted below. In addition, the Pool to which the selected Job or Jobs will be migrated is defined by the
+To run a Migration job, you must first define a Job resource very similar to a Backup Job but with :config:option:`dir/job/Type`\  = Migrate instead of :config:option:`dir/job/Type`\  = Backup. One of the key points to remember is that the Pool that is specified for the migration job is the only pool from which jobs will be migrated. In addition, the Pool to which the selected Job or Jobs will be migrated is defined by the
```

Comment:
```suggestion
To run a Migration job, you must first define a Job resource very similar to a Backup Job but with :config:option:`dir/job/Type`\  = Migrate. In Migration job the Pool defined serve as a source for all jobs that need to be migrated, implicitly defining the storage used for reading volumes. The :config:option:`dir/pool/NextPool`\  =\ ... defines the target pool to which the job will be written. The storage used as a writer is deducted from this `NextPool` definition.

```

Reply:
I'd rather keep the original wording, mentioning the detail of the read storage selection is not necessary here when explaining the concept.